benchexec/benchexec.py

```diff
@@ -140,7 +140,7 @@ def create_argument_parser(self):
                             dest="walltimelimit", default=None,
                             help='Wall time limit for each run, e.g. "90s" '
                                  '(overwrites wall time limit from XML file, '
-                                 'use "-1" to disable time limits completely)',
+                                 'use "-1" or the default value to use CPU time limit plus a few seconds))',
```

Comment:
What is meant by "or the default value" here?

And actually, the "overwrites wall time limit from XML file" is also wrong because there is no wall-time limit in the XML file.

So I guess it would be best to write this explanation from scratch in a meaningful way for users.

Reply:
I have added an option to XML format too, so this should be correct.

Is it Ok to make the following description: "Wall time limit for each run, e.g. "90s" (overwrites wall time limit from XML file, 'use "-1" to use CPU time limit plus a few seconds, such value is also used by default)" ?

Reply:
I don't see the limit in the XML? But I also think that it is not that important, fixing the documentation would be enough for me.